samples/bluetooth/direction_finding_connectionless_rx/src/main.c

```diff
@@ -0,0 +1,381 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stddef.h>
+#include <errno.h>
+#include <zephyr.h>
+
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/direction.h>
+
+#define DEVICE_NAME CONFIG_BT_DEVICE_NAME
+#define DEVICE_NAME_LEN (sizeof(DEVICE_NAME) - 1)
+#define PEER_NAME_LEN_MAX 30
+#define TIMEOUT_SYNC_CREATE K_SECONDS(10)
+
+static struct bt_le_per_adv_sync_param sync_create_param;
+static struct bt_le_per_adv_sync *sync;
+static bt_addr_le_t per_addr;
+static volatile bool per_adv_found;
+static bool scan_enabled;
+static uint8_t per_sid;
+
+static K_SEM_DEFINE(sem_per_adv, 0, 1);
+static K_SEM_DEFINE(sem_per_sync, 0, 1);
+static K_SEM_DEFINE(sem_per_sync_lost, 0, 1);
+
+#if defined(CONFIG_BT_CTLR_DF_ANT_SWITCH_RX)
+/* Example sequence of antenna switch patterns for antenna matrix designed by
+ * Nordic. For more information about antenna switch patterns see README.rst.
+ */
+static const uint8_t ant_patterns[] = { 0x2, 0x0, 0x5, 0x6, 0x1, 0x4,
+					0xC, 0x9, 0xE, 0xD, 0x8, 0xA };
+#endif /* CONFIG_BT_CTLR_DF_ANT_SWITCH_RX */
+
+static inline uint32_t adv_interval_to_ms(uint16_t interval);
```

Comment:
I think you could remove all those function prototypes. They don't seem necessary here.

Reply:
Prototypes removed.